.rubocop.yml

```diff
@@ -13,3 +13,12 @@ Style/DocumentationMethod:
 
 Style/ClassAndModuleChildren:
   Enabled: false
+
```

Comment:
I haven't seen them for a long time. Personally I feel like "something must be missing" - but that's only my personal opinion.
A good argument for me is, that the ruby style guide is a "institution" in ruby community. Everybody is able to read through them and everyone will have the same "picture" on how to write good code. We should only differ from them if there is a good reason for.